### ISSUE TITLE
[Bug] rich align not work

### ISSUE THREAD

**Issue by yaohong0713** [apache/echarts#19740]
### Version

5.5.0

### Link to Minimal Reproduction

option = {   title: {     text: 'Referer of a Website',     subtext: 'Fake Data',     left: 'center'   },   tooltip: {     trigger: 'item'   },   textStyle: {     rich: {       orgname: {         fontSize: 12,         width: 100,         align:'right'       },       count: {         fontSize: 12,         verticalAlign: 'top',         align: 'center',         color: '#597FF3',         padding: [0, 0, 0, 15],       }     }   },   legend: {     orient: 'vertical',     left: 'left',     formatter:(name) => {                         const arr = [                `{orgname|${name}}`,                `{count|100}`,             ];            return arr.join('');         }   },   series: [     {       name: 'Access From',       type: 'pie',       radius: '50%',       data: [         { value: 1048, name: 'Search Engine' },         { value: 735, name: 'Direct' },         { value: 580, name: 'Email' },         { value: 484, name: 'Union Ads' },         { value: 300, name: 'Video Ads' }       ],       emphasis: {         itemStyle: {           shadowBlur: 10,           shadowOffsetX: 0,           shadowColor: 'rgba(0, 0, 0, 0.5)'         }       }     }   ] };

### Steps to Reproduce

Just run it directly

### Current Behavior

![7baf8bc083da3bd1212ffccc8b33afd](https://github.com/apache/echarts/assets/23545051/eef61fe7-8e3e-4042-b2fa-563148ca8caf)

### Expected Behavior

Display text to the right

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
yes, second _rich_ format spoils the width of the first. Otherwise alignment works - [Demo](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=PYBwLglsB2AEC8sDeAoWszGAG0iAXMhgE4QDmZApsYQOQRiUC2tsAvgDRqzaVXQATQqnTpgpStDB0AbtUgBjAIbZaXUT0oAzabFq8da7ui3imSsIxp6k4stCVNKAHyQOnbWAEYADD7ZGGowAHmAAymAAnrzCxqKkCgAWsRoadu6UwrCmUmEQAF6Z3gBMHLAA7hACYMklPmUq5NB0pGSJYKyccRoKwACuUlm92OJ0AMQArACcAOwAYnMAzGqwIEoCAhDQZIQA2vWwBwdeEwC67N0XGmzcXegAztQQlPd7cSKpGXQAggoKL_dYHNiMAWOpUlEQEVaCBnoFUsR1hA-q89BMfABSeEaAQWJRvVKiJCwGQqPpFXwAFgAHGUvnowpQlMQkrAAKLbLaUTrgwnE0nYcmEGaLCZ0xzQgAiEGIlAUHXYvNS_LJRQm1IO9NobPMEFUisu6BVgqKNMp4qcdAAqtAoHBvgJ7jzDUQBULYIs_BboQA1KqUYCwB1Oq6pU5K9DMECJJT3CCoj6EhjMCLRIqJwmwe4xgTAcoAIUF1l8EdS2fWeYA8lotI8wAANQj1F0PHN5gDCOFGemIZAARkoABRHMpHAB0EwAlLQXTdCXPRAvTig2ABuIA)

![image](https://github.com/apache/echarts/assets/13038071/862b292f-4f62-4a8a-b3d2-b6c825d3de11)

**Comment by yaohong0713** [comment]
Thank you for your answer，But if I have to use a style for the second rich text，the first align not work
![116b4c6ab6dc42321683ea1e325e5c7](https://github.com/apache/echarts/assets/23545051/4b1d9f13-320a-4447-8143-c4e48c20ade7)

**Comment by yaohong0713** [comment]
and i tried textAlign instead of align，Still not working

**Comment by natee** [comment]
@Ovilia  any updates on this? the rich label align 'left' seems doesn't work.

[official demo](https://echarts.apache.org/examples/zh/editor.html?c=doc-example%2Ftitle-block&edit=1&reset=1)
